onerepo.config.ts

```diff
@@ -0,0 +1,59 @@
+import type { Config } from 'onerepo';
+import { changesets } from '@onerepo/plugin-changesets';
+import { docgen } from '@onerepo/plugin-docgen';
+import { eslint } from '@onerepo/plugin-eslint';
+import { jest } from '@onerepo/plugin-jest';
+import { vitest } from '@onerepo/plugin-vitest';
+import { prettier } from '@onerepo/plugin-prettier';
+import { typescript } from '@onerepo/plugin-typescript';
+import { performanceWriter } from '@onerepo/plugin-performance-writer';
+
+export default {
+	root: true,
+
+	core: {
+		generate: { templatesDir: './config/templates' },
+		graph: { customSchema: './config/graph-schema.ts' },
+		tasks: {
+			ignore: ['**/README.md', '**/CHANGELOG.md', '.changeset/**'],
+		},
+	},
+	plugins: [
+		changesets(),
```

Comment:
in this new setup all the plugins have to be defined in the root onerepo config instead of the consuming app onerepo config? i.e. i thought these were only in the monorepo's onerepo.config before this change?

Reply:
They're moved from `bin/<whatever>.mjs` to here.